### ISSUE TITLE
Working on accessibility issues 

### ISSUE THREAD

**Issue by hkaur008** [mozilla/perfcompare#347]
Hey @Carla-Moz AND @kimberlythegeek,
I want to improve accessibility on the tool and finding issues for the same.
- I checked accessibility issues [accessibility tests](https://app.circleci.com/pipelines/github/mozilla/perfcompare/1103/workflows/8aaaeeb7-a7d5-422a-90fa-29257fe7c1e4/jobs/1107/parallel-runs/0/steps/0-107). 
![image](https://user-images.githubusercontent.com/56452820/224297101-c1cbd944-979c-4f98-911e-d9597128c5c9.png)

- I conducted the [WAVE Web Accessibility Evaluation Tools](https://wave.webaim.org/) on staging branch https://perfcompare-staging.herokuapp.com/.

![image](https://user-images.githubusercontent.com/56452820/224295371-6f3d5e0f-6ef1-422f-95f8-b478e2b364b3.png)

Tasks on which I want to work 

- [ ]  ✕ SelectedRevisionsTable should have no violations (1118 ms)
- [ ]  ✕ SearchResultsList should have no violations (829 ms)
- [ ]  X Missing form label
![image](https://user-images.githubusercontent.com/56452820/224297586-08747da8-f86d-4ef6-a57d-a6e2f9c5ae81.png)

@Carla-Moz @kimberlythegeek  Please assign this issue me and guide me little regarding accessibility improvement.
However I want undertake other error as well in subsequent pull requests.

**Comment by MuhamamdShehzad** [comment]
@kimberlythegeek I would also like to work on this issue. Kindly assign it to me.
